src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Access modifier for the `Lottery` class is missing. It's a good practice to explicitly define the access level, even if it's the default.